src/main/java/net/fabricmc/network/impl/IndexedChannel.java

```diff
@@ -64,12 +64,12 @@ public void sendToServer(AbstractPacket packet) {
 
 //	Server -> Client(s)
 	public void sendToPlayer(AbstractPacket packet, EntityPlayerServer player) {
```

Comment:
@shadowfacts can you check I updated the mappings correctly, Im not 100% sure this is correct.